packages/react-composites/src/composites/common/ParticipantContainer.tsx

```diff
@@ -1,25 +1,36 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 import React from 'react';
-import { participantListStack, participantListStyle, participantListWrapper, listHeader } from './styles/Chat.styles';
-import { OnRenderAvatarCallback, ParticipantList, ParticipantMenuItemsCallback } from '@internal/react-components';
+import {
+  participantListStack,
+  participantListStyle,
+  participantListWrapper,
+  listHeader
+} from '../ChatComposite/styles/Chat.styles';
+import {
+  OnRenderAvatarCallback,
+  ParticipantList,
+  ParticipantListProps,
+  ParticipantMenuItemsCallback
+} from '@internal/react-components';
 import { FocusZone, Stack } from '@fluentui/react';
 import { useLocale } from '../localization';
-import { usePropsFor } from './hooks/usePropsFor';
-import { AvatarPersona, AvatarPersonaDataCallback } from '../common/AvatarPersona';
+import { usePropsFor } from '../ChatComposite/hooks/usePropsFor';
```

Comment:
This is a bit of a red flag, that code from `common` is calling `ChatComposite/hooks/usePropsFor`.

`usePropsFor` from there might only work for chat components.  Looking at how this is used below, I'd recommend that `ParticipantContainerProps.participatnListProps` be a required field, that is populated with `usePropsFor` from inside `ChatComposite` and `MeetingComposite`.

Reply:
okay I will clean this up!